networking/portfwd.go

```diff
@@ -233,7 +233,7 @@ func (e *podEnv) portFwdChain(name string) string {
 func (e *podEnv) portFwdChainRuleSpec(chain string, name string) []string {
 	switch name {
 	case "SNAT":
-		return []string{"-s", "127.0.0.1", "!", "-d", "127.0.0.1", "-j", chain}
+		return []string{"!", "-d", "127.0.0.1", "-j", chain}
```

Comment:
The source was set here as well as in the chain... different masquerade rules might have different `dstIPHost`'s within the same chain though so seems best just to remove it from this rule.